test/extended/testdata/build-secrets/test-docker-build.json

```diff
@@ -25,17 +25,17 @@
     },
     "strategy":{
       "type":"Docker",
-      "env": [
-        {
-          "name": "BUILD_LOGLEVEL",
-          "value": "5"
-        }
-      ],
```

Comment:
seems like as part of these changes we ought to have at least one test that confirms you can set the build loglevel via the env variable in the buildconfig defintion (perhaps just a variant on the --build-loglevel test case we already have)